gradle/libs.versions.toml

```diff
@@ -23,7 +23,6 @@ androidx-core-ktx = { module = "androidx.core:core-ktx", version.ref = "corektxV
 androidx-appcompat = { module = "androidx.appcompat:appcompat", version.ref = "appcompatVersion" }
 android-google-material = { module = "com.google.android.material:material", version.ref = "materialDesignVersion" }
 
-kotlin-stdlib = { module = "org.jetbrains.kotlin:kotlin-stdlib", version.ref = "kotlin" }
```

Comment:
As mentioned in the PR blurb, the Kotlin Gradle plugin adds the Kotlin stdlib for you so you don't need to add this yourself.